### ISSUE TITLE
Commander agent run specific target agent from the Control Target List

### ISSUE THREAD

**Issue by kkawatra09** [huginn/huginn#2937]
Is there a way to control specific target agent from the control target list using the commander agent?

**Comment by virtadpt** [comment]
I don't think so, no.  This would best be done with multiple Commander Agents and conditional actions.
